src/ucp/api/ucp.h

```diff
@@ -3395,8 +3395,10 @@ ucs_status_ptr_t ucp_get_nb(ucp_ep_h ep, void *buffer, size_t length,
  *                                progress of the operation. The application is
  *                                responsible for releasing the handle using
  *                                @ref ucp_request_free "ucp_request_free()" routine.
+ * 
+ * @note The only datatype ucp_dt_make_contig(1) is supported.
```

Comment:
`Only the datatype ucp_dt_make_contig(1) is supported for @a param->datatype, see @ref ucp_dt_make_contig`

Reply:
reworded